Mathlib/Topology/PartialHomeomorph.lean

```diff
@@ -26,8 +26,17 @@ instead of `e.toFun x` and `e.invFun x`.
 * `PartialHomeomorph.trans`: the composition of two partial homeomorphisms
 * `PartialHomeomorph.refl`: the identity partial homeomorphism
 * `PartialHomeomorph.ofSet`: the identity on a set `s`
+* `PartialHomeomorph.restr s`: restrict a partial homeomorphism `e` to `e.source ∩ interior s`
 * `PartialHomeomorph.EqOnSource`: equivalence relation describing the "right" notion of equality
-  for partial homeomorphisms
+for partial homeomorphisms
+* `PartialHomeomorph.prod`: the product of two partial homeomorphisms,
+as a partial homeomorphism on the product space
```

Comment:
Shouldn’t you indent those lines? GitHub’s syntax highlighting seems to think so, and it would also be nicer to read.